pom.xml

```diff
@@ -98,6 +98,12 @@
 			<version>2.0.0.Final</version>
 			<scope>provided</scope>
 		</dependency>
+                <dependency>
+                    <groupId>javax.ejb</groupId>
+                    <artifactId>ejb-api</artifactId>
+                    <version>3.0</version>
+                    <scope>provided</scope>
+		</dependency>
```

Comment:
This is quite a change which changes the current dependency requirements of OmniFaces.

Is this absolutely necessary?

Reply:
This is to import `@Remove` and `@Local` annotations. The only other way I know how to do it is to import the javaee-api dependency.